ai_services/anomaly_detection/README.md

```diff
@@ -1,15 +1,36 @@
+comprehensive set of APIs to help developers upload raw data, train the anomaly 
+detection model using their own business-specific data, and detect anomalies from the 
+stored model. AD service takes care of underline infrastructure and scaling 
+requirements, freeing up developers to focus on creating applications and solutions to 
+achieve their business objectives. 
+Note: You can quickly explore OCI Anomaly Detection Service in OCI LiveLabs 
+[Get Started with OCI Anomaly Detection Workshop](https://apexapps.oracle.com/pls/apex/dbpm/r/livelabs/view-workshop?wid=819&clear=180&session=1329701598349)
 
-<b>Ready-to-go results</b>
+Although it is very easy to configure and use AD service, users must ensure that input data 
+meets AD’s [Input Data requirements](https://docs.oracle.com/en-us/iaas/Content/anomaly/using/data-require.htm#data_require).
+In many instances, users are advised to use [OCI 
+Data Integration](https://docs.oracle.com/en-us/iaas/data-integration/home.htm) (DIS) or [OCI Data Flow](https://www.oracle.com/big-data/data-flow/) (DF) to transform, pre-process and convert data 
+into to meet AD’s [input specifications](https://docs.oracle.com/en-us/iaas/Content/anomaly/using/data-require.htm#data_require), which can be a bit of learning curve for AD users. 
+This is where this Git Repo can be very helpful. This repo provides multiple examples of 
+common AD pre-processing steps using DIS and DF, that can be used by users as 
+templates to build end-to-end solution.
 
-OCI Anomaly Detection outputs include identified anomalies, ML model-based estimated values, and anomaly scores. Developers use these results to assess the severity of identified anomalies and automate business workflows to address them immediately.
+[DF:  Common Pre-Processing Workflow used in Anomaly Detection](data_preprocessing_examples/oci_data_flow_based_examples)
+[DIS: Common Pre-Processing Workflow used in Anomaly Detection](data_preprocessing_examples/oci_data_integration_based_examples)
 
-<b>Easy to integrate and deploy</b>
+### When should I use DF vs DIS?
 
-OCI Anomaly Detection is a multitenant service over public REST APIs. Developers can deploy a scalable anomaly detection service easily without in-house data science and ML support, all with the lowest cost platform for data networking, storage, and egress.
-
-<b>Scalability on demand</b>
-
-OCI Anomaly Detection automatically scales for training and detection needs across all data sources and loads. Developers can now focus on creating applications and solutions to achieve their business goals, without worrying about the infrastructure.
\ No newline at end of file
+While both DF and DIS are sophisticated platforms that can be used to address you pre-processing needs,
+ you may find DF performing faster for large datasets(more than 100k observations or so). Also, DF is able to handle more columns, and certain tasks like batching.
```

Comment:
For DF we can further add: "and certain tasks like batching" -> "and various data preprocessing tasks such as batching, sharding, pivoting on large datasets in a distributed fashion."